.circleci/config.yml

```diff
@@ -942,10 +938,18 @@ commands:
       - attach_workspace:
           at: /go/src/github.com/stackrox/rox
 
+      - run:
+          name: Ensure product branding is set
+          command: |
+            if [[ -z "${ROX_PRODUCT_BRANDING}" ]]; then
+              echo >&2 "Will not proceed until product branding env variable is defined"
```

Comment:
It would be better to put "Will not proceed until ROX_PRODUCT_BRANDING env variable is set to " so the user knows what to set.

Reply:
Makes sense. Updated.